dali/tensorflow/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 if (BUILD_TENSORFLOW)
-
+  collect_headers(DALI_INST_HDRS PARENT_SCOPE)
   file(GLOB tmp *.cc)
```

Comment:
why not use collect sources for those `*.cc`?

Reply:
Done